doc/users/prev_whats_new/whats_new_3.3.0.rst

```diff
@@ -185,83 +153,210 @@ for out-of-range values with colors that differ from adjacent in-range colors.
+set with the new rcParameter :rc:`axes.titley`.
+
+.. plot::
+
+    fig, axs = plt.subplots(1, 2, constrained_layout=True, figsize=(5, 2))
+    axs[0].set_title('y=0.7\n$\sum_{j_n} x_j$', y=0.7)
+    axs[1].set_title('y=None\n$\sum_{j_n} x_j$')
+    plt.show()
+
+
+Dates now use a modern epoch
+----------------------------
+
+Matplotlib converts dates to days since an epoch using `.dates.date2num` (via
+`matplotlib.units`). Previously, an epoch of ``0000-12-31T00:00:00`` was used
+so that ``0001-01-01`` was converted to 1.0. An epoch so distant in the
+past meant that a modern date was not able to preserve microseconds because
+2000 years times the 2^(-52) resolution of a 64-bit float gives 14
+microseconds.
+
+Here we change the default epoch to the more reasonable UNIX default of
+``1970-01-01T00:00:00`` which for a modern date has 0.35 microsecond
+resolution.  (Finer resolution is not possible because we rely on
+`datetime.datetime` for the date locators). Access to the epoch is provided
+by `~.dates.get_epoch`, and there is a new :rc:`date.epoch` rcParam. The user
+may also call `~.dates.set_epoch`, but it must be set *before* any date
+conversion or plotting is used.
+
+If you have data stored as ordinal floats in the old epoch, a simple
+conversion (using the new epoch) is::
+
+    new_ordinal = old_ordinal + mdates.date2num(np.datetime64('0000-12-31'))
+
+
+tight_layout now supports suptitle
+----------------------------------
+
+Previous versions did not consider `.Figure.suptitle`, and so it may overlap
+with other artists after calling `~.Figure.tight_layout`:
+
+.. plot::
+
+    fig, axs = plt.subplots(1, 3)
+    for i, ax in enumerate(axs):
+        ax.plot([1, 2, 3])
+        ax.set_title(f'Axes {i}')
+
+    t = fig.suptitle('suptitle')
+    t.set_in_layout(False)
+    fig.tight_layout()
+
+From now on, the ``suptitle`` will be considered:
+
+.. plot::
+
+    fig, axs = plt.subplots(1, 3)
+    for i, ax in enumerate(axs):
+        ax.plot([1, 2, 3])
+        ax.set_title(f'Axes {i}')
+
+    fig.suptitle('suptitle')
+    fig.tight_layout()
+
+
+Allow tick formatters to be set with str or function inputs
+-----------------------------------------------------------
+
+`~.Axis.set_major_formatter` and `~.Axis.set_minor_formatter`
+now accept `str` or function inputs in addition to `~.ticker.Formatter`
+instances. For a `str` a `~.ticker.StrMethodFormatter` is automatically
+generated and used. For a function a `~.ticker.FuncFormatter` is automatically
+generated and used.
+
+
+Setting axes box aspect
+-----------------------
+
+It is now possible to set the aspect of an axes box directly via
+`~.Axes.set_box_aspect`. The box aspect is the ratio between axes height
+and axes width in physical units, independent of the data limits.
+This is useful to, e.g., produce a square plot, independent of the data it
+contains, or to have a usual plot with the same axes dimensions next to
+an image plot with fixed (data-)aspect.
+
```

Comment:
have a usual plot is a bit ambigous

Reply:
I think it means non-image plot, but I'll let @tacaswell confirm.